Packs/Palo_Alto_Networks_Enterprise_DLP/Integrations/Palo_Alto_Networks_Enterprise_DLP/Palo_Alto_Networks_Enterprise_DLP.py

```diff
@@ -205,31 +304,259 @@ def test(client):
         raise DemistoException("Integration test failed: Unexpected status ({})".format(status_code))
 
 
+def print_debug_msg(msg: str):
+    """
+    Prints a message to debug with QRadarMsg prefix.
+    Args:
+        msg (str): Message to be logged.
+
+    """
+    demisto.debug(f'PAN-DLP-Msg - {msg}')
+
+
+def update_incident(client: Client, incident_id: str, feedback: str, user_id: str, region: str,
+                    report_id: str, dlp_channel: str):
+    feedback_enum = FeedbackStatus[feedback.upper()]
+    result_json, status = client.update_dlp_incident(incident_id, feedback_enum, user_id, region, report_id, dlp_channel)
+
+    output = {
+        'feedback': feedback_enum.value,
+        'success': status == 200
+    }
+    if feedback_enum == FeedbackStatus.EXCEPTION_GRANTED:
+        minutes = result_json['expiration_duration_in_minutes']
+        output['duration'] = minutes / 60
```

Comment:
Is it possible there won't be a duration?

Reply:
Not really but I still added a check before doing the math.